gnfish/get_combined_seqs.py

```diff
@@ -119,16 +82,91 @@ def combine_all_sequences(path, data_type):
-            combine_all_sequences(path + "/../Data/" + data_type.capitalize() + "/*", data_type)
-    if not found:
-        logger.error("You must use genomic, rna or directory arguments.")
+def main():
+    # Arguments
+    parser = argparse.ArgumentParser(
+        description="Combines FASTA files in just one matrix"
+    )
+    parser.add_argument(
+        "output_name",
+        help="name for the file of the combined matrix",
+        nargs="?",
+        type=str,
+    )
+    parser.add_argument(
+        "output_path", help="path were to store the combined matrix.", type=str
+    )
+    parser.add_argument(
+        "--genomic",
+        help="looks at ./Data/Genomic.",
+        nargs="?",
+        const="genomic",
+        type=str,
+    )
+    parser.add_argument(
+        "--rna", help="looks at ./Data/Rna.", nargs="?", const="rna", type=str
+    )
+    parser.add_argument(
+        "--protein",
+        help="looks at ./Data/Protein.",
+        nargs="?",
+        const="protein",
+        type=str,
+    )
+    parser.add_argument("--directory", help="sets path to custom folder.", type=str)
+    parser.add_argument(
+        "--pattern",
+        help="By default, the pattern for file searching is “final.fas”. As back up to look for “final_translated.fas”, “RAW.fas”, and “RAW_translated.fas”.",
+        nargs="?",
+        default="final.fas",
+        const="final.fas",
+        type=str,
+    )
+    parser.add_argument(
+        "--out_exten",
+        help='extension of the output file. Default "_all_combined.fas".',
+        nargs="?",
+        type=str,
+        default="_all_combined.fas",
+    )
+    args = parser.parse_args()
+    output_name = args.output_name
+    output_path = args.output_path
+    genomic = args.genomic
+    rna = args.rna
+    protein = args.protein
+    directory = args.directory
+    pattern = args.pattern
+    # pattern = re.sub("\.fas", "", pattern)
+    out_exten = args.out_exten
+    path = os.getcwd()
+    data_type_lst = [genomic, rna, protein]
+    print(directory)
+
+    if directory is not None:
+        for data_type in data_type_lst:
+            logger.info(f"Running directory and {data_type} argument.")
+            combine_all_sequences(
+                directory + "/*",
+                data_type,
+                pattern,
+                output_path,
+                output_name,
+                out_exten,
+            )
+    else:
+        found = False
+        for data_type in data_type_lst:
+            if data_type is not None:
+                found = True
+                logger.info(f"Running {data_type} argument.")
+                combine_all_sequences(
+                    path + "/../Data/" + data_type.capitalize() + "/*",
+                    data_type,
+                    pattern,
+                    output_path,
+                    output_name,
+                    out_exten,
+                )
+        if not found:
```

Comment:
I think you can remove the `if not found` and have the logger in the `else` block